Automattic/vip-go-mu-plugins
# Jetpack 10.6 default update

## Description
Jetpack 10.6 is the default version of Jetpack  
  * Unpinned sites will be switched to using Jetpack 10.6
  * Changed doc block meta to reflect 10.6

## Changelog Description
### Plugin Updated: Jetpack 10.6
The default version of Jetpack has been upgraded to 10.6

## Checklist

## Steps to Test
1. Check out PR.
1. Go to `wp-admin` > `Jetpack`
1. Verify version is 10.6
